### ISSUE TITLE
[FEAT] Exclude Strapi and Third party plugins from navigation settings

### ISSUE THREAD

**Issue by SalahAdDin** [VirtusLab-Open-Source/strapi-plugin-navigation#430]
### Summary

Following [the issue](https://github.com/pluginpal/strapi-plugin-sitemap/issues/167) from **strapi-plugin-sitemap**, when we are choosing the content types we want to include on the navigation menus, we see some extra content types we should not see there:
![ExtraContentTypeNavigation](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/5034215/c6a8eca7-d98b-4796-bf6d-aadc9a48f5f2)

### Why is it needed?

As those content types are related to internal Strapi API and third-party plugins, they should not be on the selection, they should be excluded by default. 

### Suggested solution(s)
The plugin should focus on the API content types and exclude other content types, so the admin user would not see such strange content types.

As you can see from the comment `yarn strapi content-types:list `, the only content types we want to see are the following:
```shell
├──────────────────────────────────────────────┤
│ api::about-me.about-me                       │
├──────────────────────────────────────────────┤
│ api::category.category                       │
├──────────────────────────────────────────────┤
│ api::education.education                     │
├──────────────────────────────────────────────┤
│ api::post.post                               │
├──────────────────────────────────────────────┤
│ api::profile.profile                         │
├──────────────────────────────────────────────┤
│ api::project.project                         │
├──────────────────────────────────────────────┤
│ api::tag.tag                                 │
├──────────────────────────────────────────────┤
│ api::work-experience.work-experience         │
└──────────────────────────────────────────────┘
```

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request.

**Comment by CodeVoyager** [comment]
Hi, @SalahAdDin 

PR addressing your request is available in related section of this issue details. Since there are some use cases for developers to pick content types of other plugins or strapi content types, I introduced an option to pick you, as a developer, are interested in custom content types only. So this new option is backward compatible.

Hope this feature will cover all your needs.

**Comment by SalahAdDin** [comment]
> Hi, @SalahAdDin
> 
> PR addressing your request is available in related section of this issue details. Since there are some use cases for developers to pick content types of other plugins or strapi content types, I introduced an option to pick you, as a developer, are interested in custom content types only. So this new option is backward compatible.
> 
> Hope this feature will cover all your needs.

Did you documented it? It didn't know anything about that.

**Comment by cyp3rius** [comment]
Released in `2.5.2`
